### PR TITLE
Add instructions of how to extract the version of the repo

### LINKED ISSUE #263: Add version number detail to `bug_report.yml`
Once we work out how to get the version, let's update the issue templates in #262

### PR DESCRIPTION
**Pull request by paddyroddy**
This will need testing once #471 is merged. Closes #263.

Can't think of a nice way to make this Windows friendly.

### CODE REVIEW

**Review by samcunliffe — APPROVED** [review]
Tag v1.0.0 (then test this) then merge?

**Comment by samcunliffe** [comment]
(Don't think this needs two reviewers)

**Review by samcunliffe — CHANGES_REQUESTED** [review]
Ah, doesn't quite work as desired:
```
❯ git -C "$HOME/.cookiecutters/python-tooling" describe --tags --abbrev=0

latest
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -36,17 +36,17 @@ body:
...
-  #   validations:
-  #     required: true
+  - id: version
+    type: input
+    attributes:
+      label: Version In Use
+      description: |-
+        Can be found by
+        ```sh
+        git -C "$HOME/.cookiecutters/python-tooling" describe --tags --abbrev=0
```
Comment (samcunliffe) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
        git -C "$HOME/.cookiecutters/python-tooling" describe --match "v*" --tags --candidates 1
```

**Comment by paddyroddy** [comment]
> Ah, doesn't quite work as desired:
> 
> ```
> ❯ git -C "$HOME/.cookiecutters/python-tooling" describe --tags --abbrev=0
> 
> latest
> ```

Ah probably because the `latest` tag is made afterwards 😞

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (paddyroddy) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: I've tested this and it seems to work fine

**Review by samcunliffe — APPROVED** [review]
:blueberries: :ninja:

**Pull request merged**